extensions/parquet/base/src/main/java/io/deephaven/parquet/base/BulkWriter.java

```diff
@@ -81,7 +84,8 @@ int writeBulkVector(@NotNull final BUFFER_TYPE bulkValues,
      * @return a {@link WriteResult} containing the statistics of the result.
      */
     @NotNull
-    WriteResult writeBulkFilterNulls(@NotNull BUFFER_TYPE bulkValues, int rowCount);
+    WriteResult writeBulkFilterNulls(@NotNull BUFFER_TYPE bulkValues,
```

Comment:
Just for clarity, do you think we can rename this method
```suggestion
    WriteResult writeBulkVectorFilterNulls(@NotNull BUFFER_TYPE bulkValues,
```

Reply:
Thanks